cifar10/main.py

```diff
@@ -136,7 +138,9 @@ def run(
     nproc_per_node=None,
     with_pbar=False,
     with_amp=False,
-    rescaled_size=None,
+    with_cutmix=False,
```

Comment:
Let's use `cutmix_beta > 0` as a flag to activate cutmix. No need to introduce `with_cutmix`.

Reply:
I do not have the will to argue abot that :D